### PR TITLE
fix the location of env-specific Travis WDIO configfile

### PR DESCRIPTION
**Pull request by AronNovak**
Referring to:
https://github.com/Gizra/drupal-elm-starter/pull/43#pullrequestreview-25502021

### CODE REVIEW

File: client/wdio.conf.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ exports.config = {
         // 5 instances get started at a time.
         maxInstances: 5,
         //
-        browserName: 'chrome'
+        browserName: 'firefox'
```
Comment (amitaibu) [inline client/wdio.conf.js]: keep on chrome

File: client/wdio.conf.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ exports.config = {
     // Services take over a specific job you don't want to take care of. They enhance
     // your test setup with almost no effort. Unlike plugins, they don't add new
     // commands. Instead, they hook themselves up into the test process.
-    services: ['browserstack', 'selenium-standalone'],
+    services: ['browserstack', 'selenium-standalone', 'phantomjs'],
```
Comment (amitaibu) [inline client/wdio.conf.js]: remove `phantomjs`

File: client/wdio.conf.travis.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    //     },
+    //     webdriverrtc: {},
+    //     browserevent: {}
+    // },
+    //
+    // Test runner services
+    // Services take over a specific job you don't want to take care of. They enhance
+    // your test setup with almost no effort. Unlike plugins, they don't add new
+    // commands. Instead, they hook themselves up into the test process.
+    services: ['browserstack', 'selenium-standalone'],
```
Comment (amitaibu) [inline client/wdio.conf.travis.js]: this can be just `services: ['selenium-standalone'],`

**Comment by AronNovak** [comment]
Committed. Just for my curiosity, we do use browserstack in non-Travis environments, right? So for the other config file, it totally makes sense to keep it, doesn't it?

**Comment by amitaibu** [comment]
> we do use browserstack in non-Travis environments

For now, no

**Pull request merged**